src/ImageLoaders/Pef/PefFile.cs

```diff
@@ -0,0 +1,104 @@
+using Reko.Core;
+using Reko.Core.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Reko.ImageLoaders.Pef
+{
+    public class PefFile
+    {
+        public readonly PefContainer Container;
+        public PefLoaderInfo LoaderInfo;
+
+        private PefImageSegment[] pefSegments;
+
+        private PefFile(PefContainer container, PefImageSegment[] imageSegments)
+        {
+            this.Container = container;
+            this.pefSegments = imageSegments;
+        }
+
+        private PefLoaderInfo ProcessLoaderSection(PefImageSegment loaderSegment)
+        {
+            var pefLoaderSegment = PefLoaderSegment.Load(loaderSegment);
+            var loaderInfo = PefLoaderInfo.Load(pefLoaderSegment);
+            return loaderInfo;
+        }
+
+        private Address GetAddress(int sectionIndex, uint sectionOffset)
```

Comment:
PRobably should add a comment or clarify the intent of this method. I would guess it is `Obtain an address from a offset into a section.`